### PR TITLE
Add vendor code to product supplierinfo

### PR DESCRIPTION
**Pull request by cgrigis**

### CODE REVIEW

**Comment by cgrigis** [comment]
@robinkeunen Voici finalement ma PR. :)

File: spp_custom/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -3,3 +3,4 @@ Specifics customizations for SPP
 * buttons to generate product's default codes
 * set default inventory filter to 'partial', i.e. 'Sélectionner les produits manuellement'
 * add skills field
+* add vendor code to product supplierinfo
```
Comment (robinkeunen) [inline spp_custom/readme/DESCRIPTION.rst:6]: top, bien vu 👌

File: spp_custom/__manifest__.py [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "data": [
         "data/product_sequence.xml",
         "views/product.xml",
+        "views/product_supplierinfo_vendor_code_view.xml",
```
Comment (robinkeunen) [inline spp_custom/__manifest__.py]: For file naming, we try to follow the [oca conventions](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming). That would be here: `views/product_supplierinfo_views.xml`.

\* to be fair, we mostly name according to `views/<model>.xml` but we're _tending_ towards oca conventions ;-)

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Small naming comment, otherwise, looks good. I'll deploy it on our test servers.

File: spp_custom/__manifest__.py (hunk 2 above)
Comment (cgrigis) [reply spp_custom/__manifest__.py]: Excellent, I was indeed struggling to decide how to name the file, so thank you for pointing out the reference. :)

**Review by robinkeunen — APPROVED** [review]
LGTM. Just waiting Denis or Virginie's test feedback.

**Pull request merged**